### ISSUE TITLE
Optimize readability of the tracking list

### ISSUE THREAD

**Issue by solakram** [fstenstrom/world-boss-timers#112]
![image](https://github.com/fstenstrom/world-boss-timers/assets/8371481/8b3fb02d-702e-4a61-9357-7359d41d7660)

the layout, seperation and font kinda makes it hard to read. or at least it could be a lot better.

possible enhancements:

- instead of seperating the data with a ":", use a more "table like" appraoch (aka each column as a fixed width, with nice spacing)
- use a better time format (f.e. 00:00 - 12:00)
-- maybe also add hours, aka 04:00:00 - 08:00:00)
- maybe use a font that has the same width for each letter, but "table like" styling should handle this well enough

here is a bad photoshop as example:
![Untitled](https://github.com/fstenstrom/world-boss-timers/assets/8371481/d465d0da-9b33-4cd2-b81e-6c6acacd8054)

im not an UI/UX expert, so there may be other and better solutions for a better UX ;)

**Comment by fstenstrom** [comment]
I agree that here is room for improvement. The main issue is that there is no built-in monospace font. It's probably possible to add one as a resource but it's not something I've looked into.

Improving to 00:00-12:00 instead of 0s-2m 0s is definitely possible.
